### DISCUSSION TITLE
inventree-lectronz - Integrating the Lectronz Marketplace into InvenTree

### DISCUSSION THREAD

**Discussion by 30350n** [inventree/InvenTree#6974] [Show and tell]
Hi everyone! I'm using InvenTree for my inventory management and have been selling some products on [Lectronz](https://lectronz.com/) (basically a way better version of Tindie, focused on open-source hardware).

To simplify my workflow, I've been working on [`inventree-lectronz`](https://github.com/30350n/inventree_lectronz), a InvenTree plugin which offers integration for the Lectronz Marketplace.

It features:

- [x] **Automatic order synchronization**
  *For each new Lectronz order the plugin will automatically create a InvenTree Sales Order.*
- [x] **Automatic order fulfillment**
  *Lectronz orders will be automatically fulfilled if you complete a Sales Order Shipment.*

v1.0 is available now (I've been using it for the past few weeks and haven't been running into any issues anymore).

**Comment by SchrodingersGat** [comment]
@30350n very cool plugin! It is stable enough for us to feature on the website?

**Reply by 30350n** [reply]
I hope so ^^. Though I can't really guarantee much yet, as I've been the only user afaik.

**Reply by SchrodingersGat** [reply]
Feel free to submit a PR to [the website](https://github.com/inventree/inventree-website) to increase visibility for the plugin!
